packages/components/Button/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui-react-native/button",
   "version": "0.5.21",
-  "description": "A cross-platform opinionated Fluent Text component",
+  "description": "A cross-platform Button component using the FluentUI design system",
```

Comment:
@FalseLobster  I've been yelled at for this: it's Fluent Design System. "FluentUI Design System doesn't exist." :'(